templates/vue/.gitignore

```diff
@@ -6,6 +6,7 @@ config.js
 cypress.json
 cypress/screenshots/
 cypress/videos/
+.env
```

Comment:
`*.env` might be better

Reply:
@narendrasss @wynnset I realized that the `.gitignore` in master actually has this entry already, looks like this branch hasn't been updated in a hot minute.